autode/config.py

```diff
@@ -1,163 +1,176 @@
+    # NCI and Reactant and Product complex conformers. Total number of
+    # conformers will be:
+    #   (num_complex_sphere_points ×
+    #              num_complex_random_rotations) ^ (n molecules in complex - 1)
     #
     num_complex_sphere_points = 10
-    # ----------------------------------------------------------------------------------------------
-    # Number of random rotations of a molecule that is added to a NCI or Reactant/Product complex;
-    # larger numbers will be slower, but more likely to find the minimum
+    # -------------------------------------------------------------------------
+    # Number of random rotations of a molecule that is added to a NCI or
+    # Reactant/Product complex
     #
     num_complex_random_rotations = 10
-    # ----------------------------------------------------------------------------------------------
-    # For more than 2 molecules in a complex the conformational space explodes, so limit the maximum
-    # number to this value
+    # -------------------------------------------------------------------------
+    # For more than 2 molecules in a complex the conformational space explodes,
+    # so limit the maximum number to this value
     #
     max_num_complex_conformers = 300
-    # ----------------------------------------------------------------------------------------------
+    # -------------------------------------------------------------------------
 
     class ORCA:
-        # ------------------------------------------------------------------------------------------
-        # Parameters for orca                        https://sites.google.com/site/orcainputlibrary/
-        # ------------------------------------------------------------------------------------------
+        # ---------------------------------------------------------------------
+        # Parameters for orca   https://sites.google.com/site/orcainputlibrary/
+        # ---------------------------------------------------------------------
         #
-        # Note: path can be unset and will be assigned if it can be found in $PATH
+        # Path can be unset and will be assigned if it can be found in $PATH
         path = None
 
-        keywords = Keywords(low_opt=['LooseOpt', 'PBE', 'RI', 'D3BJ', 'def2-SVP', 'def2/J'],
-                            grad=['EnGrad', 'PBE', 'RI', 'D3BJ', 'def2-SVP', 'def2/J'],
-                            opt=['Opt', 'PBE0', 'RIJCOSX', 'D3BJ', 'def2-SVP', 'def2/J'],
-                            opt_ts=['OptTS', 'Freq', 'PBE0', 'RIJCOSX', 'D3BJ', 'def2-SVP', 'def2/J'],
-                            hess=['Freq', 'PBE0', 'RIJCOSX', 'D3BJ', 'def2-SVP', 'def2/J'],
-                            optts_block='%geom\nCalc_Hess true\nRecalc_Hess 30\nTrust -0.1\nMaxIter 150\nend',
-                            sp=['SP', 'PBE0', 'RIJCOSX', 'D3BJ', 'def2/J', 'def2-TZVP'])
+        loose_opt = ['LooseOpt', 'PBE', 'RI', 'D3BJ', 'def2-SVP', 'def2/J']
```

Comment:
Why is ORCA written individually first, then made into KeywordsSet, while the others aren't?

Reply:
Was a line length thing - will change to be more consistent